README.md

```diff
@@ -4,10 +4,142 @@
 **Josh Abbott**
 
 ## Craft Voxel Engine Tank Game
+This will add a tank minigame to this Minecraft clone. When completed the user can type into the console "/play tank minigame" 
+and this will spawn the user as a tank into an arena to battle an opposing enemy tank using missile blocks. The terrain of the arena is
```

Comment:
after "/play tank minigame" I would get rid of the "and" and just have a comma there but besides that it looks good.